docs/pyproject.md

```diff
@@ -61,40 +73,76 @@ More identifiers are listed at the [SPDX Open Source License Registry](https://s
 If your project is proprietary and does not use a specific licence, you can set this value as `Proprietary`.
 {{% /note %}}
 
+```toml
+license = "MIT"
+```
+
 ## authors
 
 The authors of the package. **Required**
 
 This is a list of authors and should contain at least one author. Authors must be in the form `name <email>`.
 
+```toml
+authors = [
+    "Sébastien Eustace <sebastien@eustace.io>"
```

Comment:
Please add a trailing comma -- it's generally TOML best practice when you split a list over multiple lines like this.